File: src/main/java/baboon/industry/block/storage/entity/TileEntityTransformerHVtoMV.java
package baboon.industry.block.storage.entity;

import baboon.industry.IndustryConfig;

public class TileEntityTransformerHVtoMV extends TileEntityTransformerBase {

    public TileEntityTransformerHVtoMV() {
        setCapacity(IndustryConfig.cfg.getInt("Energy Values.mvIO"));
        setTransfer(IndustryConfig.cfg.getInt("Energy Values.mvIO"));
        setMaxReceive(IndustryConfig.cfg.getInt("Energy Values.hvIO"));
    }
}

File: src/main/kotlin/turniplabs/industry/blocks/entities/transformer/TileEntityTransformerHVtoMV.kt
package turniplabs.industry.blocks.entities.transformer

import sunsetsatellite.sunsetutils.util.Connection
import sunsetsatellite.sunsetutils.util.Direction
import turniplabs.industry.IndustryConfig
import turniplabs.industry.blocks.entities.TileEntityEnergyConductorDamageable

class TileEntityTransformerHVtoMV : TileEntityEnergyConductorDamageable() {

    init {
        this.setCapacity(8192)
        this.setTransfer(IndustryConfig.cfg.getInt("Energy Values.mediumVoltage"))
        this.setMaxReceive(IndustryConfig.cfg.getInt("Energy Values.highVoltage"))


        setConnection(Direction.Y_POS, Connection.INPUT)
        setConnection(Direction.Y_NEG, Connection.OUTPUT)
    }
}